revocation/result/results.go

```diff
@@ -90,6 +159,9 @@ type CertRevocationResult struct {
 	// Thus, in the case of more than one ServerResult, this will be ResultUnknown
 	Result Result
 
+	// CRLResults is the result of the CRL check for this certificate
```

Comment:
Please provide more detailed descriptions of `CRLResults` in the comment, similar to the one of `ServerResults`. For example, when will its length be 0, 1, and/or any other number?